raiden/tests/integration/api/test_restapi.py

```diff
@@ -1064,12 +1063,37 @@ def test_api_payments_secret_hash_errors(
         json={
             'amount': amount,
             'identifier': identifier,
-            'secret_hash': secret_hash,
+            'secret': secret,
+            'secret_hash': secret,
         },
     )
     response = request.send().response
     assert_proper_response(response, status_code=HTTPStatus.CONFLICT)
 
+
+@pytest.mark.parametrize('number_of_nodes', [2])
+def test_api_payments_with_secret_no_hash(
+        api_server_test_instance,
+        raiden_network,
+        token_addresses,
+):
+    _, app1 = raiden_network
+    amount = 200
+    identifier = 42
+    token_address = token_addresses[0]
+    target_address = app1.raiden.address
+    secret = '0x78c8d676e2f2399aa2a015f3433a2083c55003591a0f3f3349b6e50fc9ca44f1'
```

Comment:
Can you use `raiden.test.utils.factories.make_secret` instead of hardcoding the value?